src/fty_alert_stats.cc

```diff
@@ -28,12 +28,28 @@
 
 #include "fty_alert_stats_classes.h"
 
+static zactor_t *alert_stats_server;
+
 static int s_resync_timer (zloop_t *loop, int timer_id, void *output)
 {
     zstr_send (output, "RESYNC");
     return 0;
 }
 
+static int s_resync_actor_pipe (zloop_t *loop, zsock_t *reader, void *arg)
+{
+    return -1;
+}
+
+static void s_resync_actor (zsock_t *pipe, void* resyncPeriod)
+{
+    zloop_t *resync_loop = zloop_new();
+    zloop_timer(resync_loop, atol((const char*)resyncPeriod) * 1000, 0, s_resync_timer, alert_stats_server);
```

Comment:
Should there be a sanity check here (that the string is good, not NULL, and `atol()` succeeds)?

Reply:
The string can't be NULL, but passing garbage will set the loop delay to 0. I'll modify the code to log an error and use the default resync period instead.